src/base/static/state/ducks/places.js

```diff
@@ -204,10 +230,27 @@ export default function reducer(state = INITIAL_STATE, action) {
         ...state,
         loadStatus: action.payload,
       };
+    case LOAD_PLACE_AND_SET_IGNORE_FLAG:
```

Comment:
This action is here to guard against the possibility that a directly-loaded Place will be duplicated when the dataset page containing that page is fetched separately. We use a piece of state to store the id of a directly-loaded Place.

An alternative would be to de-dupe the `placeModels` state every time a new array of Places is added, but this is a quadratic time operation and I think is overkill here since there is only ever one situation (direct load to a place) that would create the potential for a duplicate.